tools/find-inactive-collaborators.mjs

```diff
@@ -15,7 +15,7 @@ const args = parseArgs({
 });
 
 const verbose = args.values.verbose;
-const SINCE = args.positionals[0] || '18 months ago';
+const SINCE = args.positionals[0] || '6 months ago';
```

Comment:
```suggestion
const SINCE = args.positionals[0] || '9 months ago';
```
sounds like a good balance between the concern raised by @mhdawson and this suggestion